submissions/pseudoscience/pseudoscience_non-fiction.md

```diff
@@ -0,0 +1,29 @@
+
+### Nuclear fuel is 3 times hotter than the surface of the Sun
+
+Now it's time to come back from space and take a look at the artificial things. Locomotives have a constant energy consumption of 600 kW, but better fuel makes them faster. This means higher-tier fuel increases the engine's efficiency. Since useful power is force times velocity, efficiency is proportional to acceleration times top speed. Now, since we can use any fuel to power the locomotive, we can safely assume it is based on a [heat engine](https://en.wikipedia.org/wiki/Heat_engine). If we assume that the engine is a perfect heat engine that works using the [Carnot cycle](https://en.wikipedia.org/wiki/Carnot_cycle), and that coal burns at a temperature of 165 degrees Celsius (based on boiler steam temperature) and ambient temperature is 15 degrees, after plugging in all the numbers we get that nuclear fuel has a temperature of over 18000 degrees.
+
+### Nukes are severely underpowered
+
+One piece of uranium-235 can produce 10 fuel cells, each of which contains 8 GJ (let's not think about neighbor bonus). This means a single item of U-235 has 80 GJ of fission energy in it, and since one nuke uses 30 pieces of U-235, with a 20% fission rate (which is what you get with boosted fission something that is required for such a small nuke), it should have an explosive yield of 115 tons of TNT. That doesn't sound like a lot, but one of those would destroy everything completely in a 100 tile radius, would kill everything in a 250 tile radius, and would do damage in a 500 tile radius (based on [Nukemap](https://nuclearsecrecy.com/nukemap/))
+
+### Items have extremely low densities
+
+This part is based on [u/Maouitippitytappin](https://www.reddit.com/user/Maouitippitytappin/)'s [Reddit post](https://www.reddit.com/r/factorio/comments/j8cxvv/my_journey_to_find_scale_in_factorio/) about their journey to find scale in Factorio. To recap their post: by knowing the energy consumption of iron smelting, we can find the mass of 1 iron plate, which ends up being only around 40 grams.
+
+Knowing that, the iron needed for a single locomotive weighs a measly 6.7 kilograms, while the locomotive itself takes up a volume of about 24 m\^3 (assuming it's 2 high based on how it looks), meaning it is about 3 times less dense than air.
```

Comment:
This is actually very amusing and interesting. Feels like it deserve a bit more attention in the piece. (i.e. more longerer)